src/xlsx/conversion/format_conversion.ts

```diff
@@ -43,7 +43,7 @@ export function convertXlsxFormat(
 
       convertedFormat = convertedFormat.replace(/\\./g, (match) => match[1]); // unescape other characters
 
-      if (isXlsxDateFormat(convertedFormat)) {
```

Comment:
why remove the function? I find it more readable than reading the new if predicate